core/src/test/java/org/jdbi/v3/core/TestMultipleCommitsWithoutAutocommit.java

```diff
@@ -0,0 +1,41 @@
+package org.jdbi.v3.core;
+
+import org.assertj.core.api.Assertions;
+import org.jdbi.v3.core.junit5.H2DatabaseExtension;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class TestMultipleCommitsWithoutAutocommit {
+    @RegisterExtension
+    public H2DatabaseExtension h2Extension = H2DatabaseExtension.instance();
+    @BeforeEach
+    public void startUp() {
+        Assertions.setMaxStackTraceElementsDisplayed(100);
+    }
+
+    @Test
+    public void testAfterCommitThrowsRollback() throws Exception {
```

Comment:
Is this test method name reflecting what is going on here? I don't see any throws or rollback happening.

Reply:
Fixed